spec/factories/barcode_printer_factories.rb

```diff
@@ -1,63 +1,22 @@
 # frozen_string_literal: true
 
 FactoryBot.define do
-  # Build an API v1 barcode printer. You probably want to use: plate_barcode_printer or
-  # tube_barcode_printer instead
-  factory :barcode_printer, class: Sequencescape::BarcodePrinter, traits: [:api_object] do
-    json_root { 'barcode_printer' }
-
-    active { true } # Whether Sequencescape reports the printer as active or not
```

Comment:
Do we want to keep active?

Reply:
I don't think it's actually referred to anywhere inside Limber.  It certainly didn't break any unit tests.  I can't accurately test these changes as it stands because I still have loads of other v1 endpoints to unblock first.  Plus I can't actually print without the hardware 😆 It may be a hang over from some previous functionality.